scheduler/cupsd.h

```diff
@@ -220,3 +220,7 @@ extern void		cupsdStopSelect(void);
 /* server.c */
 extern void		cupsdStartServer(void);
 extern void		cupsdStopServer(void);
+
+extern int 		_cupsSafeStrcmp (const char *s1, const char *s2, void *data);
```

Comment:
No no no. Include cups-private.h or array-private.h to get the internal definitions, don't add duplicate prototypes